examples/test-controls/index.html

```diff
@@ -11,7 +11,7 @@
     <a-scene stats="true">
       <!-- We wrap the camera in an <a-entity> so we can position it without colliding with the HMD updates -->
       <a-entity>
-        <a-entity position="0 1.8 4" camera look-controls wasd-controls></a-entity>
+        <a-entity position="0 1.8 4" camera controls></a-entity>
```

Comment:
controls is very generic you cannot tell what this will do without looking at the implementation